build.gradle.kts

```diff
@@ -250,7 +250,8 @@ tasks {
         dependsOn(named("updateDocs"))
     }
 
-    named<Upload>("uploadArchives") {
+    register("uploadArchives") {
+        group = "upload"
         dependsOn(named("publish"))
     }
```

Comment:
can we also upgrade the node and yarn version.
node {
    version.set("14.17.5")
    yarnVersion.set("1.22.11")
    download.set(true)
}